src/popup/components/no-tickets.test.tsx

```diff
@@ -6,11 +6,7 @@ import type { Props } from "./no-tickets";
 import NoTickets from "./no-tickets";
 
 jest.mock("./error-details", () => ({ errors }: ErrorDetailsProps) => (
-  <ul>
-    {errors.map((error, index) => (
-      <li key={index}>{error.toString()}</li> // eslint-disable-line react/no-array-index-key
-    ))}
-  </ul>
+  <>{errors.map((error) => error.toString()).join("\r\n")}</>
```

Comment:
Is there a reason to use `"\r\n"` rather than just `"\n"`? Not criticizing Just curious. 🙂 

If you want to emphasize the fact that the passed errors are a list, I could imagine:

```suggestion
  <>{errors.map((error) => `- {error}`).join("\n")}</>
```

could highlight this a bit more. It doesn't have any practical impact though so no need to change.

Reply:
Hm, now that you point towards it, I actually have no clue what the `\r` does 🤷 I just quickly searched for the best and most secure way to do line breaks in `JS`. But after all, I haven't seen `\n` fail, so let's drop the `\r`.

For the bullet point part: yes, I was already thinking about the same 💚  Then let's do this!